adam-cli/src/main/scala/org/bdgenomics/adam/cli/Reads2Coverage.scala

```diff
@@ -0,0 +1,95 @@
+/**
+ * Licensed to Big Data Genomics (BDG) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The BDG licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.bdgenomics.adam.cli
+
+import org.apache.spark.SparkContext
+import org.bdgenomics.adam.projections.AlignmentRecordField._
+import org.bdgenomics.adam.projections.Projection
+import org.bdgenomics.adam.rdd.ADAMContext._
+import org.bdgenomics.adam.rdd.features.CoverageRDD
+import org.bdgenomics.adam.rdd.read.{ AlignedReadRDD, AlignmentRecordRDD }
+import org.bdgenomics.utils.cli._
+import org.kohsuke.args4j.{ Argument, Option => Args4jOption }
+
+/**
+ * Reads2Coverage (accessible as the command 'coverage' through the CLI) takes two arguments,
+ * an Read file and an output file, and calculates the number of reads from the Read file
+ * at every location in the file. Optional arguments are negativeStrands, and positiveStrands,
+ * which only save coverage computed from negative and positive strands, respectively.
+ */
+object Reads2Coverage extends BDGCommandCompanion {
+  val commandName: String = "coverage"
+  val commandDescription: String = "Calculate the depth from a given ADAM file"
+
+  def apply(cmdLine: Array[String]): BDGCommand = {
+    new Reads2Coverage(Args4j[Reads2CoverageArgs](cmdLine))
+  }
+}
+
+class Reads2CoverageArgs extends Args4jBase with ParquetArgs {
+  @Argument(required = true, metaVar = "INPUT", usage = "The reads file to use to calculate depths", index = 0)
+  var inputPath: String = null
+  @Argument(required = true, metaVar = "OUTPUT", usage = "Location to write the coverage data in ADAM/Parquet format", index = 1)
+  var outputPath: String = null
+  @Args4jOption(required = false, name = "-negativeStrands", usage = "Compute coverage for negative strands")
```

Comment:
The use case for stranded coverage is for featurization of coverage for DNase in many algorithms. These require separate featurization for negative and positive strands.